### ISSUE TITLE
[CCE] huaweicloud_cce_node_attach reports kms_key_id related error from version 1.44.0

### ISSUE THREAD

**Issue by zhenzh** [huaweicloud/terraform-provider-huaweicloud#2864]
When I upgrade my provider from 1.43.0 to 1.46.0, running terraform plan reports error below:
```
2023-03-28T10:06:39.327+0800 [TRACE] provider.terraform-provider-huaweicloud_v1.46.0: Called downstream: tf_req_id=3c59026f-f739-08cd-6264-5901709e8464 @caller=github.com/hashicorp/terraform-plugin-sdk/v2@v2.24.0/helper/schema/resource.go:1016 @module=sdk.helper_schema tf_rpc=ReadResource tf_provider_addr=provider tf_resource_type=huaweicloud_cce_node_attach timestamp=2023-03-28T10:06:39.327+0800
2023-03-28T10:06:39.328+0800 [TRACE] provider.terraform-provider-huaweicloud_v1.46.0: Received downstream response: @module=sdk.proto tf_req_duration_ms=848 tf_resource_type=huaweicloud_cce_node_attach @caller=github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/internal/tf5serverlogging/downstream_request.go:37 tf_proto_version=5.3 tf_provider_addr=provider tf_req_id=3c59026f-f739-08cd-6264-5901709e8464 tf_rpc=ReadResource diagnostic_error_count=1 diagnostic_warning_count=0 timestamp=2023-03-28T10:06:39.327+0800
2023-03-28T10:06:39.328+0800 [ERROR] provider.terraform-provider-huaweicloud_v1.46.0: Response contains error diagnostic: @caller=github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/internal/diag/diagnostics.go:55 diagnostic_summary="Error setting CCE Node fields: 1 error occurred:
        * Invalid address to set: []string{"root_volume", "0", "kms_key_id"}

" tf_proto_version=5.3 tf_provider_addr=provider tf_resource_type=huaweicloud_cce_node_attach @module=sdk.proto diagnostic_detail= diagnostic_severity=ERROR tf_req_id=3c59026f-f739-08cd-6264-5901709e8464 tf_rpc=ReadResource timestamp=2023-03-28T10:06:39.327+0800
2023-03-28T10:06:39.328+0800 [TRACE] provider.terraform-provider-huaweicloud_v1.46.0: Served request: tf_resource_type=huaweicloud_cce_node_attach @module=sdk.proto tf_provider_addr=provider tf_req_id=3c59026f-f739-08cd-6264-5901709e8464 tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/tf5server/server.go:761 tf_proto_version=5.3 timestamp=2023-03-28T10:06:39.328+0800
2023-03-28T10:06:39.328+0800 [TRACE] vertex "huaweicloud_cce_node_attach.cce_node_attach[\"heydaizi-1\"]": visit complete
2023-03-28T10:06:39.328+0800 [TRACE] dag/walk: upstream of "root" errored, so skipping
2023-03-28T10:06:39.330+0800 [TRACE] vertex "huaweicloud_cce_node_attach.cce_node_attach": dynamic subgraph encountered errors: Error setting CCE Node fields: 1 error occurred:
        * Invalid address to set: []string{"root_volume", "0", "kms_key_id"}

,Error setting CCE Node fields: 1 error occurred:
        * Invalid address to set: []string{"root_volume", "0", "kms_key_id"}

,Error setting CCE Node fields: 1 error occurred:
        * Invalid address to set: []string{"root_volume", "0", "kms_key_id"}

,Error setting CCE Node fields: 1 error occurred:
        * Invalid address to set: []string{"root_volume", "0", "kms_key_id"}

,Error setting CCE Node fields: 1 error occurred:
        * Invalid address to set: []string{"root_volume", "0", "kms_key_id"}
2023-03-28T10:06:39.330+0800 [TRACE] vertex "huaweicloud_cce_node_attach.cce_node_attach": visit complete
2023-03-28T10:06:39.330+0800 [TRACE] vertex "huaweicloud_cce_node_attach.cce_node_attach (expand)": dynamic subgraph encountered errors: Error setting CCE Node fields: 1 error occurred:
        * Invalid address to set: []string{"root_volume", "0", "kms_key_id"}

,Error setting CCE Node fields: 1 error occurred:
        * Invalid address to set: []string{"root_volume", "0", "kms_key_id"}

,Error setting CCE Node fields: 1 error occurred:
        * Invalid address to set: []string{"root_volume", "0", "kms_key_id"}

,Error setting CCE Node fields: 1 error occurred:
        * Invalid address to set: []string{"root_volume", "0", "kms_key_id"}

,Error setting CCE Node fields: 1 error occurred:
        * Invalid address to set: []string{"root_volume", "0", "kms_key_id"}
2023-03-28T10:06:39.330+0800 [TRACE] vertex "huaweicloud_cce_node_attach.cce_node_attach (expand)": visit complete
```

I have tried other provider versions, all provider after 1.43.0 have such issue.

**Comment by niuzhenguo** [comment]
@Jason-Zhang9309 pls have a look at this.

**Comment by Jason-Zhang9309** [comment]
@zhenzh this problem is fixed in 1.47.0 and later versions
